CHANGELOG.md

```diff
@@ -113,6 +113,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed deprecated `utilities.distributed.rank_zero_{warn/deprecation}` ([#10451](https://github.com/PyTorchLightning/pytorch-lightning/pull/10451))
 
 
+- Removed custom `AllGatherGrad` implementation in favor of PyTorch's ([#10504](https://github.com/PyTorchLightning/pytorch-lightning/pull/10504))
```

Comment:
Could you move this line to "Deprecated" section above and follow ananthsub's comment?

Reply:
Ok, let me change this.